docs/sources/tempo/traceql/metrics-queries.md

```diff
@@ -97,7 +100,7 @@ down by HTTP route.
 This might let you determine that `/api/sad` had a higher rate of erroring
 spans than `/api/happy`, for example.
 
-### The `count_over_time`, `min_over_time` and `max_over_time` functions
+### The `count_over_time`, `min_over_time`, `max_over_time` and `avg_over_time` functions
 
 The `count_over_time()` let you counts the number of matching spans per time interval.
```

Comment:
Should this be duration over time functions heading? 
```suggestion
### The `over_time` functions

Metrics queries supports `count_over_time`, `min_over_time`, `max_over_time`, and `avg_over_time` functions. 

The `count_over_time()` let you counts the number of matching spans per time interval.
```

Reply:
It's not only for duration, any numerical value is ok.